src/tracing/errors.ts

```diff
@@ -25,6 +25,10 @@ export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event:
 
     // If the error is based on a user rejecting, it should not be considered an exception.
     if (didUserReject(error)) return null
+
+    // Cloudflare may serve HTML error pages (eg from a 499), which are already raised as ChunkLoadError.
```

Comment:
I think @zzmp wanted to standardize multiline developer docs with `/* some comment here */` instead of `// some comment here`